tests/e2e-leg-9/verify-vrep/09-create-data-users-tlsconfig.yaml

```diff
@@ -61,7 +61,7 @@ data:
       echo "Assertion failed: expected 99, got $result"
       exit 1
     fi
-    result=$(kubectl exec $POD_NAME -i -c server -- bash -c "vsql -U dbadmin -w superuser -tAc \"SELECT COUNT(*) FROM tables;\"")
+    result=$(kubectl exec $POD_NAME -i -c server -- bash -c "vsql -U dbadmin -w superuser -tAc \"SELECT COUNT(*) FROM tables WHERE table_name = 'test_table';\"")
```

Comment:
For these `WHERE table_name = 'test_table'` do we just add them as a general precaution or we're aware of some changes / future changes that will make `tables` not empty at the start of the test?

Reply:
No my CI runs failed without them. They were picking up the internal tables that have been recently added.

Reply:
Yeah I remember seeing some emails reminding devs of this change

Reply:
Then, is this the new change from the server side? It is not related to the change in sandboxsubcluster_reconciler, right? @jizhuoyu, do you know the title of the email

Reply:
I think timing is involved. Last nights CI run was fine. But when I pulled last nights image down on my machine and ran the test I saw the issue.

Reply:
I'm wondering whether this might be potentially related to Alex's email "Potential SF impact: internal tables have been merged"